packages/snap-preact/src/Snap.tsx

```diff
@@ -229,9 +229,26 @@ export class Snap {
 				this.logger.setMode(LogMode.DEVELOPMENT);
 				this.logger.warn(`...loading build... '${branchParam}'`);
 
+				//lets try and get the siteId from the current bundle script in case its not the same as the config.
+				let siteId;
+				try {
+					const scriptsrc = document.getElementById('searchspring-context').getAttribute('src');
```

Comment:
Lets use the same query selector & logic that the getContent function uses to retrieve the script: 
https://github.com/searchspring/snap/blob/867740160cd7df24ff126c3d2068a4d5551d352f/packages/snap-toolbox/src/getContext/getContext.ts#L7-L14